### PR TITLE
feat(service-portal): Signee view for parliamentary signature collection

### PR DESCRIPTION
**Pull request by juni-haukur**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
	- Updated button navigation in the Done form for improved user flow.
	- Introduced a new SignedList component for managing signed documents, including unsigning functionality.
	- Enhanced SigneeView component to display user-specific data and improve role-based rendering.
	- Dynamic rendering in SignatureListsParliamentary component based on user ownership status.
	- OwnerView component now accepts currentCollection prop for better data handling.
	- Added `isPresidential` field to the current collection query for enhanced data context.

- **Bug Fixes**
	- Updated terminology for the Parliamentary collection type for clarity.

- **Improvements**
	- Enhanced loading indicators and user feedback throughout various components.

### CODE REVIEW

File: libs/service-portal/signature-collection/src/screens/Parliamentary/OwnerView/index.tsx [hunk 1]
```diff
@@ -14,10 +14,27 @@ import { useLocale } from '@island.is/localization'
...
+  currentCollection: SignatureCollection
+}) => {
   const navigate = useNavigate()
   const { formatMessage } = useLocale()
+  const { userInfo: user } = useAuth()
+  const { listsForOwner, loadingOwnerLists } = useGetListsForOwner(
+    currentCollection?.id || '',
+  )
+
+  console.log(listsForOwner)
```
Comment (albinagu) [inline libs/service-portal/signature-collection/src/screens/Parliamentary/OwnerView/index.tsx:37]: remove

File: libs/service-portal/signature-collection/src/screens/Parliamentary/index.tsx [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+  const { currentCollection, loadingCurrentCollection } =
+    useGetCurrentCollection()
+
   return (
     <Box>
-      <Box display={'flex'}>
-        <IntroHeader
-          title={formatMessage(m.pageTitleParliamentary)}
-          intro={formatMessage(m.pageDescriptionParliamentary)}
-          serviceProviderTooltip={formatMessage(m.infoProviderTooltip)}
```
Comment (albinagu) [inline libs/service-portal/signature-collection/src/screens/Parliamentary/index.tsx:17]: afhverju eða þessu?

Comment (albinagu) [reply libs/service-portal/signature-collection/src/screens/Parliamentary/index.tsx:17]: þeas `serviceProviderTooltip` og `serviceProviderSlug`

File: libs/service-portal/signature-collection/src/lib/constants.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export const CollectionType = {
   Presidential: 'Forsetakosningar',
-  Parliamentary: 'Alþingiskosningar',
+  Parliamentary: 'Alþingiskosning NÝTT TEST',
```
Comment (kksteini) [inline libs/service-portal/signature-collection/src/lib/constants.ts:3]: Why this value? Just curious

File: libs/service-portal/signature-collection/src/screens/Parliamentary/SignedList/index.tsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+      },
+    },
+  })
+
+  const onUnSignList = async () => {
+    try {
+      await unSign().then(({ data }) => {
+        if (
+          (
+            data as any as {
```
Comment (kksteini) [inline libs/service-portal/signature-collection/src/screens/Parliamentary/SignedList/index.tsx:39]: Can you do as unknown?
Will this work?
```suggestion
            data as unknown as {
```

Gotta keep those linter warnings down weeee :dancing_men:

File: libs/service-portal/signature-collection/src/lib/constants.ts (hunk 3 above)
Comment (juni-haukur) [reply libs/service-portal/signature-collection/src/lib/constants.ts:3]: Því þetta er valuið sem er að koma frá Þjóðskrá - það er verið að gera string comparison á "name" 🫠

File: libs/service-portal/signature-collection/src/screens/Parliamentary/SignedList/index.tsx (hunk 4 above)
Comment (juni-haukur) [reply libs/service-portal/signature-collection/src/screens/Parliamentary/SignedList/index.tsx:39]: yes I can, yes this works

File: libs/service-portal/signature-collection/src/lib/constants.ts (hunk 3 above)
Comment (juni-haukur) [reply libs/service-portal/signature-collection/src/lib/constants.ts]: refactoraði - sæki og nota núna isPresidential flaggið á currentCollection

File: libs/service-portal/signature-collection/src/screens/Parliamentary/index.tsx (hunk 2 above)
Comment (juni-haukur) [reply libs/service-portal/signature-collection/src/screens/Parliamentary/index.tsx:17]: óvart, set það aftur inn

**Pull request merged**